apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -93,9 +94,16 @@ function BaseTemplateEditorPage() {
 }
 
 export default function TemplateEditorPage() {
-  return (
-    <TemplateEditorFormProvider>
-      <BaseTemplateEditorPage />
-    </TemplateEditorFormProvider>
-  );
+  const [searchParams] = useSearchParams();
+  const type = searchParams.get('type');
+
+  if (!type || type !== 'ECHO') {
```

Comment:
This behaviour does the conditional rendering of the new vs old experience using the same routes